src/window.py

```diff
@@ -78,7 +79,7 @@ def __init__(self, **kwargs):
                       func_a_to_b=self._update_focus_and_default)
         bind_property(self.model, 'download-dir-abs',
                       func_a_to_b=self._update_success_msg)
-        self.success_msg_wdg.connect('activate-link', self._on_activate_link)
+        self.open_dir_wdg.connect('clicked', self._on_activate_link)
```

Comment:
Instead use ``<property name="action-name">app.open-download-dir</property>`` in **window.ui**.